backupstoragelocation.md

```diff
@@ -28,7 +28,7 @@ spec:
   config:
     # The AWS region where the bucket is located. Queried from the AWS S3 API if not provided.
     #
-    # Optional.
+    # Optional if s3ForcePathStyle is false.
```

Comment:
If I understand correctly, it should be optional if the s3ForcePathStyle is true?

Reply:
It should imo but see https://github.com/vmware-tanzu/velero/issues/4305#issuecomment-956936536

Reply:
The reason it can ever be optional at all is that there is a code to query region for aws. Otherwise you hit errors per op linked issue. This PR should either be merged, or the issue reopened.
https://github.com/vmware-tanzu/velero-plugin-for-aws/blob/60eb709839c0f70fb38d8c8eb4573a28834e32fa/velero-plugin-for-aws/helpers.go#L29-L31